manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go

```diff
@@ -3,6 +3,9 @@ package miqtools
 import (
 	"context"
 
+	"strconv"
```

Comment:
Any reason these moved?

Reply:
Line swapping is happening during file save so kept it like that.

Reply:
That doesn't sound like a good reason to move it and more of a local editor issue on your side.